services/QuillLMS/app/controllers/api/v1/active_activity_sessions_controller.rb

```diff
@@ -8,11 +8,12 @@ def show
   end
 
   def update
+    ActionController::Parameters.permit_all_parameters = true
```

Comment:
This allows arbitrary hashes to be accepted from the incoming payload. More details here: https://api.rubyonrails.org/v5.0.0/classes/ActionController/Parameters.html

Reply:
This is a global setting change, not just for this action, so I think we need a different way to do this.
<img width="1022" alt="Screen Shot 2022-02-24 at 3 24 43 PM" src="https://user-images.githubusercontent.com/1304933/155601898-13c892bc-aaf8-4096-815f-aefe8903cde2.png">

Reply:
Thanks for catching that - looking into a workaround